eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/persistence/model/MeetingRoom.java

```diff
@@ -0,0 +1,37 @@
+package hu.flowacademy.meetingorganizer.persistence.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Table(name = "meeting_rooms")
+public class MeetingRoom {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    private Integer numberOfSeats;
+
+    private Boolean projector;
+
+    @ManyToOne
+    private Building building;
+
```

Comment:
[minor] Töröljük ezeket a felesleges üres sorokat.

Reply:
Ha bemergelhetem majd lesz erre egy külön issue a formatterrel, hogy a többiben is meg legyen csinálva.